### PR TITLE
Add input limit on every input field

### LINKED ISSUE #4863: set input limit on ascii armored key and other input fields
references:

https://github.com/FlowCrypt/flowcrypt-security/issues/232
https://github.com/FlowCrypt/flowcrypt-security/issues/234
https://github.com/FlowCrypt/flowcrypt-security/issues/235
https://github.com/FlowCrypt/flowcrypt-security/issues/236
https://github.com/FlowCrypt/flowcrypt-security/issues/239
https://github.com/FlowCrypt/flowcrypt-security/issues/244
https://github.com/FlowCrypt/flowcrypt-security/issues/245

### PR DESCRIPTION
**Pull request by martgil**
This PR adds an input limit on every input field: generally in input and textarea elements.

close #4863 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by martgil** [comment]
The goal is to set input limits on text fields in general. I find them in a form of `input:(text|password)` and `textarea` and went through all of them. If I missed input fields in a form of different elements, please let me know.

**Comment by sosnovsky** [comment]
@tomholub it seems `LGTM analysis` check can be disabled only in project settings, I haven't found any config for it in the code. Can you please check and disable it?

**Comment by tomholub** [comment]
Yes, only admin can do that. Updated to CodeQL

File: extension/chrome/settings/setup.htm [hunk 1]
```diff
@@ -240,7 +240,7 @@ <h1>Set Up FlowCrypt</h1>
       <div class="line">
         <p class="text-success small m-0" data-test="ekm-setup-user-notify">Your keys are managed with your organization's Email Key Manager.</p>
         <label class="input_passphrase_save_label hidden" data-test="input-step2ekm-save-passphrase-label">
-          <input type="checkbox" class="input_passphrase_save" value="" name="check" data-test="input-step2ekm-save-passphrase">
+          <input type="checkbox" class="input_passphrase_save" value="" name="check" data-test="input-step2ekm-save-passphrase" maxlength="256">
```
[the remark this answers is not in the document]
Comment (martgil) [reply extension/chrome/settings/setup.htm]: thank you, sir - it can be removed. thanks again!

**Review by sosnovsky — APPROVED** [review]
Looks good 👍

**Pull request merged**